include/seastar/core/future.hh

```diff
@@ -1551,7 +1551,7 @@ public:
     /// the return value of then(), itself as a future; this allows then()
     /// calls to be chained.
     ///
-    /// This member function is only available is the payload is std::tuple;
+    /// This member function is only available when the payload is std::tuple;
```

Comment:
i guess the original author meant to put "if". but either way, this is surely an improvement.